src/psyclone/configuration.py

```diff
@@ -861,7 +861,8 @@ def __init__(self, config, section):
         # a property, and 'type' is a string.
         # These values are taken from the psyclone config file.
         self._grid_properties = {}
-        Property = namedtuple("Property", "fortran type")
+        # Store the namedtuple type to create all properties with
```

Comment:
Yes, I wanted to avoid calling the tuple constructor several times (not so much for performance reasons, but ... it somehow felt wrong to create the type again all the time). Anyway, am as happy as doing this all in ``make_property`` - done.